dtschema/schemas/simple-bus.yaml

```diff
@@ -14,7 +14,7 @@ maintainers:
 
 properties:
   $nodename:
-    pattern: "^([a-z][a-z0-9\\-]+-bus|bus|soc|axi|ahb|apb)(@[0-9a-f]+)?$"
+    pattern: "^([a-z][a-z0-9\\-]+-bus|bus|soc|axi|ahb|apb|cpld)(@[0-9a-f]+(,[0-9a-f]+)?)?$"
```

Comment:
'cpld-bus' would have worked. Though you really shouldn't have what the implementation is in the nodename. Yes, we already violated that with axi, ahb, etc., but that's really to avoid a bunch of warnings on existing dts files.

For the unit-address, we should just drop any checking of it as that is up to the parent bus. So just: '...(@.+)?$'